app/(main)/(drawer)/(tabs)/sources.tsx

```
@@ -0,0 +1,206 @@
+      );
+      setMyPages(sourceData);
+      setOtherPages(sourceData);
+      setLoading(false);
+    } catch (e) {
+      Toaster.error("Something went wrong while fetching pages");
+    }
+  };
+
+  useEffect(() => {
+    getPages();
+  }, [, isFocused]);
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)